code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/NesterovAlexanderTest.java

```diff
@@ -0,0 +1,89 @@
+package ru.unn.agile.ComplexNumber.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class NesterovAlexanderTest {
+    @Test
+    public void isRightNormComplexNumber() {
+        // Arrange
+        ComplexNumber z1 = new ComplexNumber(2, 6);
+        ComplexNumber z1Inv = new ComplexNumber(2, -6);
+        double normZ = Math.pow(2, 2) + Math.pow(6, 2);
+
+        // Act
+        ComplexNumber z = z1.multiply(z1Inv);
+
+        // Assert
+        assertEquals(new ComplexNumber(normZ, 0), z);
+    }
+
+    @Test
+    public void isRightRelativeMathRelation() {
+        // Arrange
+        ComplexNumber z1 = new ComplexNumber(2.22, 4.67);
+        ComplexNumber z1Inv = new ComplexNumber(2.22, -4.67);
+
+        // Act
+        ComplexNumber z = z1.add(z1Inv);
+
+        // Assert
+        assertEquals(4.44, z.getReal() + z.getImaginary(), 1e-10);
```

Comment:
Why do you use `1e-10` constant? As I googled, double in Java provides accuracy of 5-16 digits of precision

Reply:
thanks for the comments.
done.